keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -229,6 +229,11 @@ class BytePairTokenizer(tokenizer.Tokenizer):
             and will tokenize a word with a leading space differently. Adding
             a prefix space to the first word will cause it to be tokenized
             equivalently to all subsequent words in the sequence.
+        unsplittable_tokens: list, defaults to None. A list of strings that will
```

Comment:
Maybe worth calling out these "tokens" must exist in the vocab.

Reply:
sg! I can just add that as I merge